imports/server/mediasoup.ts

```diff
@@ -807,10 +1189,14 @@ Meteor.startup(async () => {
     getLocalIPAddresses() :
     await getPublicIPAddresses();
   Logger.verbose('Discovered announceable IPs', { ips: ips.map((ip) => ip.announcedIp ?? ip.ip) });
+  const [primaryIp, ...otherIps] = ips;
+  if (!primaryIp) {
+    throw new Meteor.Error('Unable to discover any IP addresses');
```

Comment:
Does this mean we'll fail to start up the backend if you're, say, developing jolly-roger while disconnected from other networks?  That's probably a developer experience regression that I'd prefer to avoid.

Reply:
In development, we get addresses from iterating over your network devices, so presumably you'll always have a loopback in the worst case